src/components/dev-hub/breadcrumb-schema.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { useSiteMetadata } from '../../hooks/use-site-metadata';
+
+const getBreadcrumbList = (breadcrumb, siteUrl) => {
+    return breadcrumb.map(({ label, target, to }, index) => {
+        const path = target || to;
+
+        return {
+            "@type": 'ListItem',
+            position: index + 1,
+            name: label,
+            item: path === '/' ? siteUrl : siteUrl + path + '/',
+        };
+    });
+};
```

Comment:
Small nitpick, we can omit the `{}` surrounding this function body which would imply a return here:

`const getBreadcrmubList = (breadcrumb, siteUrl) => breadcrumb.map(...);`

Not a concern, this is just a bit more of short-hand